app/styles/_variables.scss

```diff
@@ -1,5 +1,5 @@
 //Font Family Variables
-$default-font: "Clear Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
+$default-font:"Fira Sans", Helvetica, Arial, sans-serif;
```

Comment:
@pdehaan: Ryan and were talking about nuking Clear Sans, just wanted to see what it would look like. this is probably not going to survive the final version of this PR